src/core/utils/options.ts

```diff
@@ -6,6 +6,32 @@ import { logger } from "./logger";
 
 let userDefinedOptions: SWACLIConfig = {};
 let configFileDefinedOptions: SWACLIConfig = {};
+const commandSpecificOptions = {
+  init: ["yes"],
+  start: [
+    "appLocation",
+    "outputLocation",
+    "apiLocation",
+    "appDevserverUrl",
+    "apiDevserverUrl",
```

Comment:
not sure if this scalable.. is there a way to build this list dynamically from the these commands code?